views/js/qtiCreator/widgets/Widget.js

```diff
@@ -23,28 +23,35 @@ define([
     'taoQtiItem/qtiItem/core/Element',
     'taoQtiItem/qtiCreator/model/helper/invalidator',
     'core/logger'
-], function(_, $, Promise, Element, invalidator, loggerFactory){
+], function (_, $, Promise, Element, invalidator, loggerFactory) {
     'use strict';
 
-    var _pushState = function(widget, stateName){
-        var currentState = new widget.registeredStates[stateName](widget);
+    const _pushState = function (widget, stateName) {
+        const currentState = new widget.registeredStates[stateName](widget);
         widget.stateStack.push(currentState);
         currentState.init();
     };
 
-    var _popState = function(widget){
-        var state = widget.stateStack.pop();
-        if(state){
+    const _popState = function (widget) {
+        const state = widget.stateStack.pop();
+        if (state) {
             state.exit();
         }
     };
 
+    const createEventName = (widget, name, ns) => {
+        const eventName = `${name}.qti-widget.${widget.serial}`;
+        if (ns) {
+            return `${name}.${ns}`;
```

Comment:
Don't we need `.qti-widget` here also?

Reply:
It was there before https://github.com/oat-sa/extension-tao-itemqti/blob/11b7d1aba445245c468e4ab8be3a1cd997e7cab6/views/js/qtiCreator/widgets/Widget.js#L248
Don't want to create new regression

Reply:
you mean `${name}`? will fix

Reply:
Yes, thank you

Reply:
fixed, event name should be `${name}.qti-widget.${widget.serial}.${ns}` or `${name}.qti-widget.${widget.serial}`